clueless/models.py

```diff
@@ -33,19 +33,27 @@ class Space(models.Model):
     """
     posX = models.IntegerField()
     posY = models.IntegerField()
-    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True)
-    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True)
+    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True, null=True)
+    spaceWest = models.OneToOneField('self', related_name='spaceEast', blank=True, null=True)
     spaceCollector = models.ForeignKey(SpaceCollection)
 
+    def __str__(self):
+        return("({},{})".format(self.posX, self.posY))
+
 class Player(models.Model):
     """
     Represents a player in the context of a clueless game.  Ties back to Django user
     """
     user = models.ForeignKey(User)
     currentSpace = models.ForeignKey(Space)
-    currentGame = models.ForeignKey('Game') # game not defined yet, using string as lazy lookup
+    currentGame = models.ForeignKey('Game', blank=True, null=True) # game not defined yet, using string as lazy lookup
     character = models.ForeignKey('Character', blank=True)
 
+    def __str__(self):
+        return("user: {}, currentSpace: {}, currentGame: {}, character: {}".format(
```

Comment:
is this __ str __ a django convention? if so no worries, but if not can we rename this something better?